cmd/nerdctl/logs.go

```diff
@@ -96,6 +99,10 @@ func logsAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	stopChannel := make(chan os.Signal, 1)
+	// catch SIGETRM/SIGINT
```

Comment:
`SIGKILL` too ?

Reply:
I did update the code to include `SIGKILL` but luckily the [linter was there to stop us](https://github.com/containerd/nerdctl/actions/runs/3345763466/jobs/5541705573#step:4:29) from even trying. 😆

Reply:
I've opened a backport request at https://github.com/containerd/containerd/pull/7632 and will update go.mod when it merges, thank you both!